dev/release/download_rc_binaries.py

```diff
@@ -125,6 +140,67 @@ class Maven(Downloader):
         "/content/repositories/staging/org/apache/arrow"
 
 
+class GitHub(Downloader):
+    def __init__(self, repository, tag):
+        super().__init__()
+        if repository is None:
+            raise ValueError("--repository is required")
+        if tag is None:
+            raise ValueError("--tag is required")
+        self.repository = repository
+        self.tag = tag
```

Comment:
Should we use `_` prefix here?

```suggestion
        self._repository = repository
        self._tag = tag
```